pandas/io/formats/info.py

```diff
@@ -15,6 +15,32 @@
     from pandas.core.series import Series  # noqa: F401
 
 
+class CountConfigs(NamedTuple):
```

Comment:
Can you generate this using `collections.namedtuple` instead? We typically don't subclass things from typing

Reply:
Sure, can do, although I think this is the newer syntax, it's taken directly from https://docs.python.org/3/library/typing.html#typing.NamedTuple